src/UrlHighlightFactory.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+namespace Externals;
+
+use VStelmakh\UrlHighlight\Highlighter\HtmlHighlighter;
+use VStelmakh\UrlHighlight\UrlHighlight;
+
+class UrlHighlightFactory
+{
+    public static function createUrlHighlight(string $defaultScheme, array $attributes): UrlHighlight
+    {
+        $highlighter = new HtmlHighlighter($defaultScheme, $attributes);
+        return new UrlHighlight(null, $highlighter);
```

Comment:
Thanks for the PR!

The only thing I see that could be improved is removing that class and moving these 2 lines inside the factory in `config.php`, could you do that?

Thanks!

Reply:
That's make sense. Will do so.

Reply:
Resolved